packages/runtime/datastore-definitions/src/jsonable.ts

```diff
@@ -26,6 +22,12 @@
  *  - Non-finite numbers (`NaN`, `+/-Infinity`) are also coerced to `null`.
  *
  * Also, `Jsonable<T>` does not prevent the construction of circular references.
+ *
+ * @example
+ * Typical usage:
+ * ```typescript
+ *      function foo<T>(value: Jsonable<T>) { ... }
```

Comment:
nit: why the indentation?

Reply:
Good catch! There shouldn't be any.
```suggestion
 * function foo<T>(value: Jsonable<T>) { ... }
```